features/FEATURE_BLE/targets/TARGET_CORDIO/doc/PortingGuide.md

```diff
@@ -458,25 +413,22 @@ static void hciCoreReadResolvingListSize(void)
 }
 ```
 
-
-
 ### HCI accessor function
 
-The HCI driver is injected in the `CordioBLE` class at construction site. 
-Given that the CordioBLE class doesn't know what class shall be used to 
-construct the driver nor it knows how to construct it, the port shall provide a 
-function returning a reference to the HCI driver. 
+The HCI driver is injected to the `CordioBLE` class at manufacture.
 
-This function lives in the global namespace and its signature is: 
+Given that the `CordioBLE` class doesn't know which class constructs the driver nor how to construct it, the port provides a function returning a reference to the HCI driver.
 
-```c++
+This function is in the global namespace, and you can call it with:
```

Comment:
The following line is the signature of the function; it is the signature that must be implemented by the porting layer. 
I would like a wording indicating that it is a signature or prototype; it is not a function call.

Reply:
Sorry, I'm not really sure of the difference. Lack of technical expertise here.

Reply:
A function signature is a semantic element that defines the name of the function, the type of its parameters and what type the function returns as a result. 

If we model a function that adds two number together we can say:  
- the function is named `addition`.
- it accept two numbers in parameters.
- it returns a number as a result of the operation.

A function call is the application of the function with actual arguments. If we want to add 5 and 2 together we can call our addition function declared earlier with the numbers 5 and 2 as arguments. 

---- 

In this bit of documentation, we ask porters to create a function that construct their HCI driver. This function must be named `ble_cordio_get_hci_driver`, it has no arguments and it returns a `ble::vendor::cordio::CordioHCIDriver&`. 

In other word the function signature must be `ble::vendor::cordio::CordioHCIDriver& ble_cordio_get_hci_driver()`. 

This function is not called by the vendor port, it is called by mbed OS and that's why we define what the prototype of the function should be.

Reply:
How is this in the latest commit?